Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -1,5 +1,102 @@
 #import "BPKCalendar.h"
 
+#import <Backpack/Color.h>
+#import <Backpack/Font.h>
+#import <FSCalendar/FSCalendar.h>
+
+@interface BPKCalendar () <FSCalendarDelegate, FSCalendarDelegateAppearance, FSCalendarDataSource>
+
+@property (nonatomic) FSCalendar *calendarView;
+
+@end
+
 @implementation BPKCalendar
 
-@end
\ No newline at end of file
+- (instancetype)initWithCoder:(NSCoder *)coder
+{
```

Comment:
Can we put braces on the same line as the signature please?

Reply:
I moved them but this should be enforced by the linter.